crates/nuon/src/from.rs

```diff
@@ -7,7 +7,9 @@ use std::sync::Arc;
 
 /// convert a raw string representation of NUON data to an actual Nushell [`Value`]
 ///
-/// > **Note**
+// WARNING: please leave the following two trailing spaces, they matter for the documentation
+// formatting
+/// > **Note**  
```

Comment:
Is this a supported style for rustdoc markdown?
With those few sentences maybe we don't need special formatting here.

Reply:
adding two spaces at the end of a line will insert a newline, i.e. before it was rendering as
```
> **Note** whatever the note is
```
and now it would render as
```
> **Note**
> whatever the note is
```
as intended when the doc was first written :yum:

Reply:
'tis not critical, but these have been removed even though they carry meaning in the rendered markdown :wink: